test/Operators/integrals.jl

```diff
@@ -34,7 +34,9 @@ function test_column_integral_definite!(center_space)
     ᶜz = Fields.coordinate_field(center_space).z
     ᶠz = Fields.coordinate_field(face_space).z
     z_top = Fields.level(ᶠz, Operators.right_idx(face_space))
-    ᶜu = map(z -> (; one = one(z), powers = (z, z^2, z^3)), ᶜz)
+    ᶜu = Base.Broadcast.broadcasted(ᶜz) do z
```

Comment:
Why wouldn't `map` just work here? Won't it still call `Base.broadcast`? (same for below)

Reply:
Yes, `map` calls `Base.broadcast`. This calls `Base.Broadcast.broadcasted` to test whether the integral operators can handle unmaterialized broadcasts.

Reply:
Ah, right, missed the two letters, thanks!